packages/scroller/test/visual/lumo/scroller.test.js

```diff
@@ -0,0 +1,31 @@
+import { fixtureSync } from '@vaadin/testing-helpers';
+import { sendKeys } from '@web/test-runner-commands';
+import { visualDiff } from '@web/test-runner-visual-regression';
+import '../../../theme/lumo/vaadin-scroller.js';
+
+describe('scroller', () => {
+  let div, element;
+
+  beforeEach(() => {
+    div = document.createElement('div');
+    div.style.padding = '10px';
+
+    element = fixtureSync(
+      `
+      <vaadin-scroller style="width: 200px; height: 200px;">
+        <div id="content" style="width: 400px;"></div>
+      </vaadin-scroller>`,
+      div
+    );
+    element.firstElementChild.textContent = new Array(1000).fill('content').join(' ');
+  });
+
+  it('default', async () => {
+    await visualDiff(div, 'default');
+  });
+
+  it('focused', async () => {
```

Comment:
https://github.com/vaadin/web-components/blob/bb14747d5c1fa62c48fc9d03c2a0f7e10a3c0009/packages/text-field/test/visual/lumo/text-field.test.js#L21-L24

```suggestion
  it('focus-ring', async () => {
```

Reply:
Good catch, changed

Reply:
I suppose the screenshot name should be also updated.

Reply:
That's strange. It was renamed in https://github.com/vaadin/web-components/pull/3107/commits/584165f4be50eb7cfd6c554a94081feb737c11e5

Also the test in material was renamed :)

Reply:
Ah, the screenshots have been renamed but the test case still refers to old names.

https://github.com/vaadin/web-components/pull/3107/commits/584165f4be50eb7cfd6c554a94081feb737c11e5#diff-401597b8949165b755e5ced73c0852a52443d4380ad7c029d31303e8840e9b5aL29

Reply:
Ah, the name repeated in the `visualDiff`, now it seems to pass